templates/add-ons/documentation_starter/{{cookiecutter.__folder_name}}/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "{{ cookiecutter.title.replace(' ', '_').replace('-', '_').replace('.', '_') }}"
```

Comment:
The docs folder is a separate Python project from the add-on that is (or at least might be) in the backend folder. So it may be confusing to give them the same name. Can we add `docs` here to distinguish it?

(I don't think it shows up anywhere that matters, but I was confused for a minute why I had a "Project_Title.egg-info" folder inside the docs folder after I built the docs.)

Reply:
Done in 422c8afaa05ba815247314a2a98cc9aaceef1b4c.

Is there any way to prevent the `.egg-info` folder from getting created? It's in `.gitignore`, but documentation shouldn't have this. We could add a make target to remove it after any build, but that doesn't prevent its creation.

Reply:
I thought the .egg-info was being created when uv installs the docs project as an editable package, but I can't reproduce it now. Let's come back to this separately.

Reply:
@davisagli I tried every combination possible to reproduce this. It happens only on the first run of `uv sync`.